README.md

```diff
@@ -134,7 +134,7 @@ This section is under active development.  It will allow you to a [MITRE Map](ht
 Choose TYPE {detection, story} to create new content for the Content Pack.  The tool will interactively ask a series of questions required for generating a basic piece of content and automatically add it to the Content Pack.
 
 ### contentctl inspect
-This section is under development.  It will enable the user to perform an appinspect of the content pack in preparation for deployment onto a Splunk Instance or via Splunk Cloud.
+This section is under development.  The inspect action performs a number post-build validations. Primarily, it will enable the user to perform an appinspect of the content pack in preparation for deployment onto a Splunk Instance or via Splunk Cloud. It also compares detections in the new build against a prior build, confirming that any changed detections have had their versions incremented (this comparison happens at the savedsearch.conf level, which is why it must happen after the build).
```

Comment:
Note that I updated the README to reflect `inspect`'s expanded scope

Reply:
* Agreed on the extra wording! Good context to give a user
* As for auto-updating detection versions in YAML, I thought about that, but I think it's out of scope for this PR. It shouldn't be too heavy of a lift to add in the future, but I don't want to get bogged down in feature bloat if I can avoid it. But I think it's a great thing to get on our roadmap for ease of use; the `inspect` action could still report the error but maybe w/ an optional `--update-versions` flag it could generate the appropriate changes and the user could then commit them
* Also agreed that including the contentctl version used for build/inspect/test in an ESCU release is a great idea. I think we capture that in extracto already (at least for the version used in test, which we will soon tie to the version used in build/inspect) but having it encoded in the app is a good idea too

Reply:
@pyth0n1c updated the language in the readme

If you agree on my last two bullets above and the README looks good, I can make issues to track the two future improvements, and then resolve this convo :)

Reply:
Yup, agreed! Forgot to resolve this issue soon. Resolved.